### PR TITLE
client, daemon, cmd/snap: no more tasks; everything is changes

### PR DESCRIPTION
**Pull request by chipaca**

### CODE REVIEW

**Comment by chipaca** [comment]
retest this please

**Comment by chipaca** [comment]
retest this please

**Comment by chipaca** [comment]
Retest this please

**Comment by chipaca** [comment]
retest this please

File: daemon/api.go [hunk 1]
```diff
@@ -809,36 +783,40 @@ func sideloadSnap(c *Command, r *http.Request) Response {
...
+
+	state := c.d.overlord.State()
+	state.Lock()
+	msg := fmt.Sprintf(i18n.G("Install local %q snap"), snap)
+	chg := state.NewChange("install-snap", msg)
+	ts, err := snapstateInstall(state, snap, "", flags)
+	if err == nil {
+		chg.AddAll(ts)
+	}
+	state.Unlock()
```
Comment (pedronis) [inline daemon/api.go]: feeling a bit bad about doing this not with a defer over so much code

**Comment by pedronis** [comment]
LGTM to me, with a comment

File: daemon/api.go (hunk 1 above)
Comment (niemeyer) [inline daemon/api.go]: I'll fix that.

**Pull request merged**